.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 src/*.js
 test/*.js
+.idea/
```

Comment:
`.idea/` is developer / machine specific, a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) is preferred.

Reply:
Thank you for sharing that.